libs/scripts/delayed.js

```diff
@@ -12,12 +12,15 @@
 
 export const loadJarvisChat = async (getConfig, getMetadata, loadScript, loadStyle) => {
   const config = getConfig();
-  const jarvis = getMetadata('jarvis-chat');
-  if (!config.jarvis?.id || !config.jarvis?.version) return;
-  if (jarvis === 'on') {
-    const { initJarvisChat } = await import('../features/jarvis-chat.js');
-    initJarvisChat(config, loadScript, loadStyle);
-  }
+  const jarvis = getMetadata('jarvis-chat')?.toLowerCase();
+  if (!jarvis || !config.jarvis?.id || !config.jarvis?.version) return;
+
+  const desktopViewport = window.matchMedia('(min-width: 900px)').matches;
+  if (jarvis === 'mobile' && desktopViewport) return;
+  if (jarvis === 'desktop' && !desktopViewport) return;
```

Comment:
But there's no explicit value checking, so someone could set it to 'jarvis | testing1234' and Jarvis would be activated on that page. It's not terrible, but I'm not sure it's something we want to allow.

Reply:
Yeah it might be good to limit it to `mobile|desktop|on` and if it's not on, we just return.. we can always allow more options later but taking away the behaviour might otherwise break some sites in the future

Reply:
> someone could set it to 'jarvis | testing1234' and Jarvis would be activated on that page
That's actually acceptable, it's the same for other metadata. However, if you set `jarvis-chat|off`, then it would still be on, so I do agree that additional checks need to be performed